exercises/concept/secrets/.docs/introduction.md

```diff
@@ -65,3 +65,5 @@ Bitwise.<<<(1, 2)
 ```
 
 All bitwise functions only work on integers.
+
+If you are running Elixir version 1.9 or lower, you will need to call `require Bitwise` at the beginning of the module definition to be able to use the _Bitwise_ module.
```

Comment:
I want people solving this locally on an older Elixir version to still be able to solve it, but I don't want to explain at this point what's `require`, what's `use` and the difference between functions and macros.